From d5ad9d483cb49644c28e4499b68260ec6e600243 Mon Sep 17 00:00:00 2001
From: Assam Boudjelthia <assam.boudjelthia@qt.io>
Date: Thu, 22 Oct 2020 18:11:21 +0300
Subject: [PATCH 019/196] Android: disable Gradle caching by default

If enabled by default, the cache size will grow quickly, especially
if multiple different projects are managed every day. Keep the option
commented, which can be enabled manually by the user.

Change-Id: I6d4a67f4f1bb3e37b02de7ff4c534ec4d006f9a3
Reviewed-by: Ville Voutilainen <ville.voutilainen@qt.io>
Reviewed-by: Rami Potinkara <rami.potinkara@qt.io>
(cherry picked from commit 90912b657edf27d40aa1a8ac8fbc3bae8d285e25)
Reviewed-by: Qt Cherry-pick Bot <cherrypick_bot@qt-project.org>
(cherry picked from commit 903a3b6493f1fe66b43eb8696b706d1943230e42)
Reviewed-by: Andy Shaw <andy.shaw@qt.io>
---
 src/3rdparty/gradle/gradle.properties | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/src/3rdparty/gradle/gradle.properties b/src/3rdparty/gradle/gradle.properties
index 554f6aeba1..fded106b17 100644
--- a/src/3rdparty/gradle/gradle.properties
+++ b/src/3rdparty/gradle/gradle.properties
@@ -5,5 +5,7 @@
 # The setting is particularly useful for tweaking memory settings.
 org.gradle.jvmargs=-Xmx2048m
 
-# Enable Gradle caching
-org.gradle.caching=true
+# Gradle caching allows reusing the build artifacts from a previous
+# build with the same inputs. However, over time, the cache size will
+# grow. Uncomment the following line to enable it.
+#org.gradle.caching=true
-- 
2.25.1

